gcc/jit/libgccjit.cc

```diff
@@ -4430,7 +4430,12 @@ gcc_jit_type_set_packed (gcc_jit_type *type)
 
   type->set_packed ();
 }
-
+void
+gcc_jit_type_set_tree_addressable(gcc_jit_type *type)
+{
+  RETURN_IF_FAIL (type, NULL, NULL, "NULL type");
```

Comment:
Can this `TREE_ADDRESSABLE` attribute be used on any type?
If not, please add checks here to verify that it's OK to apply on the specified type.